.github/workflows/ci-dotnet.yml

```diff
@@ -23,4 +23,4 @@ jobs:
       java-version: 17
       os: windows
       run: |
-        bazel test //dotnet/test/common:ElementFindingTest-firefox //dotnet/test/common:ElementFindingTest-chrome --pin_browsers=true
+        bazel test //dotnet/test/common:ElementFindingTest-firefox //dotnet/test/common:ElementFindingTest-chrome
```

Comment:
Tests pass when I leave `--pin_browsers=true` in here, but it shouldn't be necessary.